### ISSUE TITLE
Publish target fails in NanoServer image

### ISSUE THREAD

**Issue by natemcmaster** [aspnet/MvcPrecompilation#168]
**Repro**
```
docker pull microsoft/dotnet-nightly:2.0.0-preview3-sdk-nanoserver
docker run --rm -it microsoft/dotnet-nightly:2.0.0-preview3-sdk-nanoserver
dotnet new web --output web
cd web
dotnet restore
dotnet publish /v:n
```

**Result**

Publication fails with:

>C:\Program Files\dotnet\sdk\NuGetFallbackFolder\microsoft.aspnetcore.mvc.razor.viewcompilation\2.0.0-rtm-26151\build\netstandard2.0\Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.targets(60,5): error MSB6004: The specified task executable location "C:\Users\ContainerAdministrator\cmd.exe" is invalid. [C:\web\web.csproj]

**Comment by natemcmaster** [comment]
Appears to be external: https://github.com/Microsoft/msbuild/issues/2273

**Comment by natemcmaster** [comment]
Verified this is fixed in MSBuild 15.3.406.
